vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_footer.scss

```diff
@@ -0,0 +1,38 @@
+/* === FOOTER */
+/* ========================== */
+
+footer {
+	background: #000;
+	color: #fff;
+	position: relative;
+	text-align: center;
+	padding-top: $gutter*1.5;
+	padding-bottom: $gutter*1.5;
+  overflow: hidden;
+	.logo-container{
+		width: 260px;
+		margin: auto;
+		.logo{
+			a{
+				color: $white
+			}
+		}
```

Comment:
@nykka Same idea as my comment above, please avoid using "parents" (`.logo`) with only one child (`a`) -- It will make your CSS code easier to read when we need to update it. 😄